geoapps_utils/run.py

```diff
@@ -163,7 +164,7 @@ def copy_uijson_relatives_only(
         with Workspace.create(workspace_path) as new_workspace:
             ifile.copy_relatives(new_workspace)
             temp_json = ifile.ui_json.copy()
-            temp_json["geoh5"] = workspace_path
+            temp_json["geoh5"] = new_workspace
```

Comment:
why new_workspace?
We don't want to store the path where the workspace is created?
Maybe it ends up doing the same thing?